lib/features/vpn/vpn_bandwidth.dart

```diff
@@ -7,9 +7,13 @@ class VPNBandwidth extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return sessionModel
-        .bandwidth((BuildContext context, Bandwidth bandwidth, Widget? child) {
-          // User does not have bandwidth cap off
-      if (bandwidth.remaining > 0 ) {
+        .bandwidth((BuildContext context, Bandwidth? bandwidth, Widget? child) {
+      return sessionModel.proUser((context, isProUser, child) {
+        if (bandwidth == null || isProUser) {
```

Comment:
This is the key change. I added an extra check here as well just to make sure we never show the data cap to Pro users